spec/requests/api/v1/clubs_request_spec.rb

```diff
@@ -182,5 +182,21 @@
         end
       end
     end
+
+    describe 'destroy: delete /club' do
+      it 'deletes a book club' do
+        club = create :club
+        delete "/api/v1/clubs/#{club.id}"
+        expect(response).to be_successful, status: 204
+        expect(response.body).to be_empty
+      end
+
+      describe 'sad path' do
+        it 'cannot find a club to delete' do
+          delete "/api/v1/clubs/999999"
```

Comment:
So this is the same issue Mal was running into. We should be expecting a 404 error here bc the club is not found. In order to get this, we need to first 'find' the club in the controller, and then try to delete that club :)